src/components/Calculator.js

```diff
@@ -1,189 +1,79 @@
 /* eslint-disable */
```

Comment:
- [ ] Please, do not disable eslint unless some exceptions are required. Kindly remove the `eslint-disable` comment.